### ISSUE TITLE
A Scene starts with no boundaries in 6.1.1

### ISSUE THREAD

**Issue by staudt** [quick-developers/quick#57]
When you create a scene, it's boundary property will start as null by default (instead of the screen size).
This will cause games from previous versions to fail to trigger the offBoundary event.

I noticed this when using the starfield plugin, it works on 6.0.x, but on 6.1.1 the stars will leave the screen and not be re-positioned because the offBoudary event is not triggered, since scene boudaries are null.

**Comment by diogoschneider** [comment]
Are you loading the library in your `<head>` block or just before your code, right before the end of the `<body>` block? Can you confirm if it happens with the very first scene passed to `Quick.init()`? And does it continue to happen if you initialize your scene contents in the `init()` method instead of the `constructor()` method? Thanks.

**Comment by diogoschneider** [comment]
Thanks for the report. The boundary property from a Scene should be usually null, since it's the boundary of the Scene, which is also a Sprite. So until we have Scenes inside Scenes, if that ever makes sense, it should remain null.

That being said, it's been previously used to provide its child Sprites with the Rect they need for a boundary, while they should actually set their boundary to the Scene itself, since it's also a Rect. To get things more confusing, the default behavior of Sprite.setBoundary() without parameters would also aim for the boundary property from the parent Scene instead of the Scene itself, which is wrong.

So this should be fixed in two steps:

Sprites which use boundaries should no longer do this:
```
sprite.setBoundary(scene.boundary);
```
But do this instead:
```
sprite.setBoundary(scene);
```
Or even:
```
sprite.setBoundary();
```
Since it should default to the parent Scene. Just make sure any of these are called on init() or after the Sprite has been added to the Scene. This should work on v6.1.2.

Also, the plugin has been fixed on v6.1.1.

**Comment by staudt** [comment]
Awesome, thanks!
